default.nix

```
@@ -0,0 +1,83 @@
+let
+  inputs = import ./flake-inputs.nix;
+in
+  {
+    nixpkgs ? inputs.nixpkgs,
```

Is there any reason we can't create the actual values here already?

```suggestion
    pkgs ? import inputs.nixpkgs { inherit system; config = {}; overlays = []; },
```

---

There's the Rust overlay, I guess passing a different `pkgs` could break `atomic-server`?

---

Though I have no idea why the overlay is there, I'll open a test PR to see what happens if it's removed.